modules/backend/classes/BackendController.php

```diff
@@ -1,14 +1,15 @@
 <?php namespace Backend\Classes;
 
-use Str;
 use App;
-use File;
-use View;
+use Closure;
 use Config;
-use Response;
+use File;
 use Illuminate\Routing\Controller as ControllerBase;
 use October\Rain\Router\Helper as RouterHelper;
-use Closure;
+use Response;
```

Comment:
Why did you change the order of imports? @w20k

Reply:
No particular reason. Already thought about that, after commit.
But for me it's just default Intellij Idea formatting, when set to auto :)

Reply:
Can you revert it? October's import statements aren't organized that way so I see no reason to change them willy nilly :)

Reply:
Sure thing :) Sorry, had a glass of wine before pull, so let it slip somehow :D

Reply:
No worries, take care of it when able :) I appreciate all your help on the project so far!

Reply:
If you could be still available in few min will make PR update ;)